### ISSUE TITLE
"Here's how you know" is not a descriptive enough link

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4033]
## Area or Component
Top USAgov header - "An official website of the United States government Here’s how you know"

## Quick Summary
The link text "here's how you know" is not descriptive enough about the link's destination

## Accessibility Violation
Name: The element's name is missing or incorrect: https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect

## Action Requested
Edit the text to be a more descriptive link (or add a better aria-label to the button directly).

Note that we might not be able to edit the button text ourselves (seems to be fairly standard across government websites) but we should be able to improve the aria-label for the button it's associated with.

## Screenshots (optional)
![Screen Shot 2022-07-21 at 5 39 00 PM](https://user-images.githubusercontent.com/80282552/180337597-7c5a43e6-d4cb-4a47-9b92-d3a52fdbe7a6.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=2:2

**Comment by emmastephenson** [comment]
aria-label example from the VA: https://developer.va.gov/
